vissl/models/__init__.py

```diff
@@ -16,7 +28,17 @@ def build_model(model_config, optimizer_config):
     The returned model is not copied to gpu yet (if using gpu) and neither
     wrapped with DDP yet. This is done later train_task.py .prepare()
     """
-    return BaseSSLMultiInputOutputModel(model_config, optimizer_config)
+    model_name = model_config.NAME
+    model_cls = get_model(model_name)
+    return model_cls(model_config, optimizer_config)
+
+
+from vissl.models.model_helpers import (  # noqa
```

Comment:
Can we move these back to the top? I would also recommend running linter: https://github.com/facebookresearch/vissl/tree/master/dev#practices-for-coding-quality